contracts/governance/extensions/GovernorVotesQuorumFraction.sol

```diff
@@ -14,10 +14,13 @@ import "../../utils/math/SafeCast.sol";
  * _Available since v4.3._
  */
 abstract contract GovernorVotesQuorumFraction is GovernorVotes {
-    using Checkpoints for Checkpoints.History;
+    using SafeCast for uint256;
+    using Checkpoints for Checkpoints.Trace224;
 
     uint256 private _quorumNumerator; // DEPRECATED
```

Comment:
This is off topic but we should add some explanation here of why this is deprecated / what it was replaced with.